<?xml version="1.0"?>
<launch>
    <arg name="roomba" default="roomba1" />
	<arg name="file_path"   default="$(env HOME)/../../media/amsl/ef88e1a9-c1cf-4f57-8b27-ab40a1bf1d68/home/amsl/bagfiles/six_roomba_spread_20220224_image" />

	<include file="$(find multi_localizer)/launch/utils/bag_player.launch" >
		<arg name="roomba"    value="$(arg roomba)" />
        <arg name="date"      value="20220224" />
        <arg name="wait_time" value="0.0" />
    </include>

	<node pkg="multi_localizer" type="image_saver" name="image_saver" >
		<remap from="img_in" to="$(arg roomba)/detected_image" />
		<param name="FILE_PATH" value="$(arg file_path)/$(arg roomba)" />
	</node>

</launch>